src/main/java/frc/robot/subsystems/swerve/SwerveDrivetrain.java

```diff
@@ -267,6 +304,66 @@ public Rotation2d[] getCancoderAngles() {
         };
     }
 
+    @SuppressWarnings("PMD.AvoidReassigningParameters")
```

Comment:
Is this still needed? I was unable to see the warning

Reply:
When borrowed from GoS, this was added, I have to ask them why they added it